vendor/adcssy.css

```diff
@@ -0,0 +1,3717 @@
+.color-primary {
```

Comment:
Specific reason why this is in vendor and not scss?

Reply:
Hmm, it's been a long time since I last dealt with css pipelines. The advantage of converting it to scss would be that unused classes would be stripped (if this even exists) and the code would be minimized. Now we would have to minify it again on build time.

Reply:
Agreed